packages/theme-addresses/__tests__/theme-addresses.test.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * @jest-environment jsdom
+ */
+import {
+  CountryProvinceSelector
+} from '../theme-addresses';
+
+const countryOptions = '<option value="---" data-provinces="[]">---</option>' +
+  '<option value="Bahamas" data-provinces="[]">Bahamas</option>' +
+  '<option value="Canada" data-provinces="[[&quot;Alberta&quot;,&quot;Alberta&quot;],[&quot;British Columbia&quot;,&quot;British Columbia&quot;],[&quot;Manitoba&quot;,&quot;Manitoba&quot;],[&quot;New Brunswick&quot;,&quot;New Brunswick&quot;],[&quot;Newfoundland&quot;,&quot;Newfoundland and Labrador&quot;],[&quot;Northwest Territories&quot;,&quot;Northwest Territories&quot;],[&quot;Nova Scotia&quot;,&quot;Nova Scotia&quot;],[&quot;Nunavut&quot;,&quot;Nunavut&quot;],[&quot;Ontario&quot;,&quot;Ontario&quot;],[&quot;Prince Edward Island&quot;,&quot;Prince Edward Island&quot;],[&quot;Quebec&quot;,&quot;Quebec&quot;],[&quot;Saskatchewan&quot;,&quot;Saskatchewan&quot;],[&quot;Yukon&quot;,&quot;Yukon&quot;]]">Canada</option>' +
+  '<option value="New Zealand" data-provinces="[[&quot;Auckland&quot;,&quot;Auckland&quot;],[&quot;Bay of Plenty&quot;,&quot;Bay of Plenty&quot;],[&quot;Canterbury&quot;,&quot;Canterbury&quot;],[&quot;Gisborne&quot;,&quot;Gisborne&quot;],[&quot;Hawke&#39;s Bay&quot;,&quot;Hawke&#39;s Bay&quot;],[&quot;Manawatu-Wanganui&quot;,&quot;Manawatu-Wanganui&quot;],[&quot;Marlborough&quot;,&quot;Marlborough&quot;],[&quot;Nelson&quot;,&quot;Nelson&quot;],[&quot;Northland&quot;,&quot;Northland&quot;],[&quot;Otago&quot;,&quot;Otago&quot;],[&quot;Southland&quot;,&quot;Southland&quot;],[&quot;Taranaki&quot;,&quot;Taranaki&quot;],[&quot;Tasman&quot;,&quot;Tasman&quot;],[&quot;Waikato&quot;,&quot;Waikato&quot;],[&quot;Wellington&quot;,&quot;Wellington&quot;],[&quot;West Coast&quot;,&quot;West Coast&quot;]]">New Zealand</option>';
+
+const shippingCountryOptions = '<option value="---" data-provinces="[]">---</option>' +
+  '<option value="France" data-provinces="[]">France</option>' +
+  '<option value="United Kingdom" data-provinces="[]">United Kingdom</option>';
+
+describe('CountryProvinceSelector', () => {
+  test('is a function exported by theme-addresses.js', () => {
+    expect(typeof CountryProvinceSelector).toBe('function');
+  });
+
+  test('returns an object with a build function', () => {
+    let countryProvinceSelector;
+    expect(() => {
+      countryProvinceSelector = new CountryProvinceSelector(countryOptions);
+    }).not.toThrow();
+    expect(typeof countryProvinceSelector.build).toBe('function');
+  });
+
+  // Hmm ... how to validate the countryOptions that is being passed in is a valid html?
```

Comment:
I would say that test belongs in the Liquid drop?

Reply:
okay